addons/sourcemod/translations/pt_p/shop.phrases.txt

```diff
@@ -229,6 +229,11 @@
 		"pt_p"		"{green}[Loja]{default} Categoria vazia!"
 	}
 
+	"MenuEmptyCategory"
+	{
+		"pt_p"		"Categoria vazia!"
+	}
+
```

Comment:
How you get this translate? Via Google Translate or you have something with knowledge of this language?

Reply:
I copied the existing translation and removed the color ids.
https://github.com/hlmod/Shop-Core/blob/aaba3bad7f56141696aecf2c7e2ac14fabd41b57/addons/sourcemod/translations/pt_p/shop.phrases.txt#L227-L230

Reply:
Why is there exclamation mark?

Reply:
> Why is there exclamation mark?

I just copied the existing translation, ask the source for such a question :D
If you need it without the `!`, then say so.